packages/ra-ui-materialui/src/layout/useSidebarState.ts

```diff
@@ -13,7 +16,12 @@ import { useStore } from 'ra-core';
  *     );
  * };
  */
-export const useSidebarState = (): useSidebarStateResult =>
-    useStore<boolean>('sidebar.open', true);
+export const useSidebarState = (): useSidebarStateResult => {
+    const isXSmall = useMediaQuery<Theme>(
```

Comment:
The sidebar was open by default on Mobile. It was not user-friendly, but most importantly it prevented e2e testing on mobile, as every test had to start by closing the menu.